credential/schema/model.go

```diff
@@ -3,28 +3,104 @@ package schema
 import "fmt"
 
 const (
-	// VCJSONSchemaType https://w3c-ccg.github.io/vc-json-schemas/v2/index.html#credential_schema_definition_metadata
-	VCJSONSchemaType string = "https://w3c-ccg.github.io/vc-json-schemas/schema/2.0/schema.json"
+	// CredentialSchema2023Type https://www.w3.org/TR/vc-json-schema/#credentialschema2023
+	CredentialSchema2023Type VCJSONSchemaType = "CredentialSchema2023"
+	// JSONSchema2023Type https://www.w3.org/TR/vc-json-schema/#jsonschema2023
+	JSONSchema2023Type VCJSONSchemaType = "JsonSchema2023"
```

Comment:
What's the difference between these two? Couldn't figure it out from the spec. I'll create an issue in the repo once I understand better the differences.

Reply:
I _think_ the difference is that the data model that dereferencing `id` returns. It's not obvious that this is the difference from looking at the definition of the `id` property, since both have the same line:

> The value is expected to be a URL that identifies the credential schema associated with the [verifiable credential](https://www.w3.org/TR/vc-json-schema/#dfn-verifiable-credential).

Reply:
yes that's the difference, since the `CredentialSchema2023` returns a `Credential` where the `credentialSubject` is a `schema` and `JsonSchema2023` just returns a `schema`